services/director-v2/src/simcore_service_director_v2/api/routes/computations.py

```diff
@@ -164,10 +165,14 @@ async def _get_project_metadata(
 
     parent_node_id = NodeID(current_project_metadata["node_id"])
     parent_node_idstr = NodeIDStr(f"{parent_node_id}")
-    parent_project_id = await project_repo.get_project_id_from_node(parent_node_id)
-    parent_project = await project_repo.get_project(parent_project_id)
-    assert parent_node_idstr in parent_project.workbench
+    try:
+        parent_project_id = await project_repo.get_project_id_from_node(parent_node_id)
+        parent_project = await project_repo.get_project(parent_project_id)
+    except (ProjectNotFoundError, ProjectNodesNodeNotFound) as exc:
+        _logger.exception("Could not find project/node: %s", f"{exc}")
```

Comment:
MINOR: no need to put `f"{exc}"` since`_logger.exception` will show the exception str and all the traceback stack

Reply:
Do you really need to log as an exception? `_get_project_metadata() ` returning `{}` because it has no parents sounds like ok to me. No need to log an error.

Reply:
@pcrespov yes. cause currently the only use of that metadata is for s4l. so in case there are metadata they should be correct. and therefore I'd like to see an error in the logs in case it happens.